docs/user-manuals/workload/daemon-pod-updater.md

```diff
@@ -0,0 +1,223 @@
+```shell
+***
+containers:
+      - name: nginx
+        image: nginx:1.19.5
+***
+```
+
+- Get pods: the old pod `default/nginx-daemonset-fhsr6` on `openyurt-e2e-test-worker` node has been deleted and the new pod `default/nginx-daemonset-slp5t` has been created; the pods on the two disconnected nodes will not be upgraded temporarily
+
+```shell
+nginx-daemonset-bv5jg   1/1     Running   0          33m     10.244.2.2   openyurt-e2e-test-worker3   <none>           <none>
+nginx-daemonset-lmmtd   1/1     Running   0          33m     10.244.3.2   openyurt-e2e-test-worker2   <none>           <none>
+nginx-daemonset-slp5t   1/1     Running   0          5m54s   10.244.1.3   openyurt-e2e-test-worker    <none>           <none>
+```
+
+- Restore network connectivity of nodes
+```shell
+$ docker network connect kind openyurt-e2e-test-worker2
+$ docker network connect kind openyurt-e2e-test-worker3
+
+$ kubectl get nodes -o wide
+NAME                              STATUS   ROLES                  AGE   VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION     CONTAINER-RUNTIME
+openyurt-e2e-test-control-plane   Ready    control-plane,master   49m   v1.22.7   172.18.0.4    <none>        Ubuntu 21.10   5.10.76-linuxkit   containerd://1.5.10
+openyurt-e2e-test-worker          Ready    <none>                 48m   v1.22.7   172.18.0.2    <none>        Ubuntu 21.10   5.10.76-linuxkit   containerd://1.5.10
+openyurt-e2e-test-worker2         Ready    <none>                 48m   v1.22.7   172.18.0.3    <none>        Ubuntu 21.10   5.10.76-linuxkit   containerd://1.5.10
+openyurt-e2e-test-worker3         Ready    <none>                 48m   v1.22.7   172.18.0.5    <none>        Ubuntu 21.10   5.10.76-linuxkit   containerd://1.5.10
+```
+
+- Get pods: daemonset pods on all nodes have been upgraded
+```shell
+nginx-daemonset-kbkf6   1/1     Running   0          88s   10.244.3.3   openyurt-e2e-test-worker2   <none>           <none>
+nginx-daemonset-scgtv   1/1     Running   0          51s   10.244.2.3   openyurt-e2e-test-worker3   <none>           <none>
+nginx-daemonset-slp5t   1/1     Running   0          11m   10.244.1.3   openyurt-e2e-test-worker    <none>           <none>
+```
+
+- Check pods image version: all pods have been upgraded to nginx:1.19.5
+```shell
+***
+Containers:
+  nginx:
+    Container ID:   containerd://f7d4b3f1257a0d1d8da862671c11cb094f9fba1ba0041b7a5f783d9c9e4d8449
+    Image:          nginx:1.19.5
+    Image ID:       docker.io/library/nginx@sha256:31de7d2fd0e751685e57339d2b4a4aa175aea922e592d36a7078d72db0a45639
+    Port:           <none>
+    Host Port:      <none>
+    State:          Running
+      Started:      Fri, 14 Oct 2022 14:21:25 +0800
+    Ready:          True
+    Restart Count:  0
+    Environment:    <none>
+    Mounts:
+      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-wrhj8 (ro)
+***
+```
+
+### 3）OTA Upgrade Model
```

Comment:
we should figure out how to enable OTA upgrade model for daemonset. plus the REST API in detail.